cogs/tags.py

```diff
@@ -841,6 +841,7 @@ async def tags(self, ctx, *, member: TagMember = None):
     def _get_tag_all_arguments(args):
         parser = Arguments(add_help=False, allow_abbrev=False)
         parser.add_argument('--text', action='store_true')
+        parser.add_argument('--json', action='store_true')
```

Comment:
This should perhaps be use [`ArgumentParser.add_mutually_exclusive_group`](https://docs.python.org/3/library/argparse.html#argparse.ArgumentParser.add_mutually_exclusive_group) as currently you can use both flags and only the first will be accepted.

Reply:
Perhaps something like this? I don't typically program in Python, so I'm sure what I'm doing can be optimized. Thank you very much for linking the docs
```python
def _get_tag_all_arguments(args):
    parser = Arguments(add_help=False, allow_abbrev=False)
    parser.add_mutually_exclusive_group(required=False)
    parser.add_argument('--text', action='store_true')
    parser.add_argument('--json', action='store_true')
    if args is not None:
        return parser.parse_args(shlex.split(args))
    else:
        return parser.parse_args([])
 ```

Reply:
`add_mutually_exclusive_group` returns the `group` that you should add the arguments to:

```py
    group = parser.add_mutually_exclusive_group(required=False)
    group.add_argument('--text', action='store_true')
    group.add_argument('--json', action='store_true')
```

Reply:
Just tested your suggested. Everything seems good, I'll put it into the branch now